src/frontends/sql/sql_ir.proto

```diff
@@ -81,21 +60,14 @@ message TagTransform {
   // The `Expression` that is having its tag state changed by the
   // `TagTransform`.
   Expression transformed_node = 1;
-  // A "map" from the unique ID of a particular input `Expression` to the
-  // integrity tags that are required to be present upon that operation for
-  // this transform to execute. Because it is possible that a `TagTransform`
-  // could require a single expression to have multiple integrity tag values,
-  // we express this `map` as a list of pairs.
-  repeated ExprTagPrecondition tag_preconditions = 2;
-  // The action to be performed on the tag state of the node indicated by the
-  // `transformed_node_id` if all of the required integrity tags are present
-  // on the nodes indicated in the `required_integrity_tags` map. Can be
-  // added or removed.
-  TagAlteration tag_alteration = 3;
```

Comment:
Don't you need to extract this information from policy rules? Are you planning to do generate Kotlin -> Datalog directly? 

In any case, the simplification in this PR is good.

Reply:
Yes, I'm planning on directly printing from Kotlin to Datalog. Initially, I was thinking about having the protos be used to communicate about rule trees and structural trees, and we would handle the policy translation on the Raksha side. However, this is probably more work than was necessary for an implementation we know will be an MVP and it was confusing my thinking about generating dataflow graph Datalog. I think this is a better separation of concerns.